docs/.vuepress/config.js

```diff
@@ -37,6 +37,12 @@ module.exports = {
     docsDir: "docs",
     editLinks: true,
     // custom text for edit link. Defaults to "Edit this page"
-    editLinkText: "Help us improve this page"
+    editLinkText: "Help us improve this page",
+    // Gets the UNIX timestamp(ms) of each file's last git commit, and it will also display at the bottom of each page in an appropriate format. The string will be displayed as a prefix.
+    lastUpdated: "Last updated",
+    // Enables a popup to refresh site content. The popup will be shown when the site is updated. It provides a refresh button to allow users to refresh contents immediately.
+    serviceWorker: {
+      updatePopup: true
```

Comment:
This sets a config value for the service worker plugin related to if it generates popups, but we don't have that plugin enabled currently.  Thus I'd recommend removing this one config from this PR, and extracting it to a separate PR to get serviceWorkers going if we want to do that, as that's slightly more involved (in addition to enabling the plugin, we would need to add a site manifest, which is required for web browsers to actually register and utilize the serviceWorker for caching.)

Reply:
ok thanks for helping me understand that. we'll just the lastUpdated config for now.